ddtrace/tracer.py

```diff
@@ -43,8 +44,8 @@ def __init__(self):
         # A hook for local debugging. shouldn't be needed or used in production
         self.debug_logging = False
 
-        # globally set tags
-        self.tags = {}
+        # globally set tags (`lang` and `lang.version` are set by default)
+        self.tags = {"lang": "python", "lang.version": platform.python_version()}
```

Comment:
maybe we should keep this separated, it would be easier to use it in the UI or to get insights later